src/endpoints/esdt/esdt.service.ts

```diff
@@ -367,7 +369,9 @@ export class EsdtService {
   }
 
   async getTokenSupply(identifier: string): Promise<EsdtSupply> {
-    const { supply, minted, burned, initialMinted } = await this.gatewayService.getEsdtSupply(identifier);
+    const esdtSupply = await this.gatewayService.getEsdtSupply(identifier);
+    this.pluginService.formatTokenSupply(identifier, esdtSupply);
```

Comment:
this should not fail if the formatTokenSupply is not defined in the plugin module

Reply:
yes, just tested and it works